test/liquid_test.rb

```diff
@@ -5,7 +5,7 @@
 require 'test_helper'
 require 'liquid/c'
 
-test_files = FileList[File.join(liquid_test_dir, 'integration/**/*_test.rb')]
+test_files = FileList[File.join(liquid_test_dir, '**/*_test.rb')]
```

Comment:
I'd prefer to keep this until we decide we need an internal incompatibility. There's missing test coverage in the integration tests that is covered by the unit tests (proc handling is the case I ran into). Since we're patching individual classes and trying to make them conform to the same API, the unit tests give more confidence of correctness.